.circleci/config.yml

```diff
@@ -176,10 +176,8 @@ jobs:
             npm link @audius/service-commands
             cd /home/circleci/project/service-commands/scripts/
             sudo "$(which node)" hosts.js add
-            sudo curl -L "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            sudo curl -L "https://github.com/docker/compose/releases/download/1.29.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
             "$(which node)" setup.js up -nc 4
-            cd /home/circleci/project/service-commands/scripts/
-            "$(which node)" setup.js run user-replica-set-manager up
             cd /home/circleci/project/mad-dog/
```

Comment:
This is now captured in the service commands up. Having this line and the up in service commands causes an error since the contract gets attempted to be redeployed.